rp_asos/tasks.py

```diff
@@ -188,7 +188,10 @@ def save_all_data_from_redcap(self, project, tz_code):
                 project, hospital, hospital_patient_records
             )
 
-    def run(self, project_id, tz_code, **kwargs):
+    def run(self, args, **kwargs):
```

Comment:
@erikh360 I think if you just change this to `*args` then you don't have to wrap the arguments in a tuple in `test_tasks.py`

Reply:
I've tested that locally, the tests pass but when I fire up a actual celery it args is `([1, 'GMT'],)`